testpilot/frontend/static-src/app/models/me.js

```diff
@@ -32,40 +32,35 @@ export default Model.extend({
   },
 
   fetch() {
+    return Promise.all([
+      this.fetchInstalledExperiments(),
+      this.fetchMeResource()
+    ]);
+  },
+
+  fetchInstalledExperiments() {
+    this.hasAddon = Boolean(window.navigator.testpilotAddon);
+    if (!this.hasAddon) { return false; }
+
+    // HACK: Previous versions of the add-on expected installation data from
+    // the server, but that's no longer available. So, let's fake it, with an
+    // empty data structure as appropriate to the addon version (or lack of one
+    // exposed)
```

Comment:
Well, it'll look to the older add-ons like the server thinks no experiments are installed. So, on a page load, it will send a PUT to the server for each installed experiment as part of the "sync" process. A few extra HTTP requests until those add-ons get updated. But, if the server already has UserInstallation records for those experiments, the requests will be idempotent no-ops.